dapps/assets/.gitignore

```diff
@@ -0,0 +1,39 @@
+# Node
+node_modules/*
+npm-debug.log
+
+# TypeScript
+src/*.js
+src/*.map
+src/*.d.ts
+
+# JetBrains
+.idea
+.project
+.settings
+.idea/*
+*.iml
+
+# VS Code
+.vscode/*
+
+# Windows
+Thumbs.db
+Desktop.ini
+
+# Mac
+.DS_Store
+**/.DS_Store
+
+# Ngc generated files
+**/*.ngfactory.ts
+
+# Build files
+dist/*
+build/*
+
+# Playground tmp files
+.playground
+.tmp
+
+documentation/
```

Comment:
I would suggest to add `"files.insertFinalNewline": true,` to your vscode config ;)

Reply:
I do have that, but it only applies `onSave` and not when I copy whole files :)
Anyhow, EOF shouldn't really matter to gitignores, but I'll add it.